user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -87,23 +99,29 @@ The elements will be created with the following specifications:
 
 1. In the lower right corner, click *Execute*.
 
-   This will open a wizard with several steps.
+   This will open a pop-up window.
+
+1. Click *Create bulk*.
 
-1. Below the **Run automation** script text, enter the path of the CSV file.
+1. Below the **Create Bulk** script text, enter the path of the CSV file.
 
    > [!NOTE]
-   > The CSV file has to have the following structure: ElementName, Be_DMA, Collector_DMA, Protocol, IpAddress, CommunityString, Network, Market, Hub, SystemUser, SystemPass.
+   >
+   > - The path has to be present in the host were the DMA is located.
```

Comment:
This isn't entirely clear. Could you explain what you mean with this?

Reply:
Hello Marieke, you are right, this can be explained better.
It could be:
The CSV file must be present on the server where the DMA is running, and its file path must be specified correctly. If this is not the case, the program will not run the desired file.